runtime/src/bank.rs

```diff
@@ -6476,16 +6490,13 @@ impl Bank {
     }
 
     pub fn fill_bank_with_ticks(&self) {
```

Comment:
nit: care to hustle to rename this to `fill_bank_with_ticks_for_tests` or something? i feared a bit that we could introduce ungated behavior changes.. i don't think this kind of code could be useful for any production code-path.